src/shared/components/post/post-listings.tsx

```diff
@@ -68,7 +68,7 @@ export class PostListings extends Component<PostListingsProps, any> {
     return (
       <div>
         {this.posts.length > 0 ? (
-          this.posts.map(post_view => (
+          this.posts.map((post_view: PostView, idx: number) => (
             <>
```

Comment:
Are the explicit types necessary? Typescript should already be able to infer these types if I'm not mistaken.

Reply:
Good point, updated!